crmd/callbacks.c

```diff
@@ -217,9 +217,9 @@ peer_update_callback(enum crm_status_type type, crm_node_t * node, const void *d
                 crm_trace("Updating CIB %s stonithd reported fencing of %s complete",
                           (down->confirmed? "after" : "before"), node->uname);
 
-            } else if (alive == FALSE) {
+            } else if ((alive == FALSE) && safe_str_eq(task, CRM_OP_SHUTDOWN)) {
                 crm_notice("%s of %s (op %d) is complete", task, node->uname, down->id);
-                /* down->confirmed = TRUE; Only stonith-ng returning should imply completion */
+                /* down->confirmed = TRUE; */
```

Comment:
Please forgive my ignorance, but why is this commented code line here, after all?